packages/url_launcher/url_launcher/example/linux/flutter/CMakeLists.txt

```diff
@@ -78,7 +78,8 @@ add_custom_command(
   COMMAND ${CMAKE_COMMAND} -E env
     ${FLUTTER_TOOL_ENVIRONMENT}
     "${FLUTTER_ROOT}/packages/flutter_tools/bin/tool_backend.sh"
-      linux-x64 ${CMAKE_BUILD_TYPE}
+      ${FLUTTER_TARGET_PLATFORM} ${CMAKE_BUILD_TYPE}
+  VERBATIM
```

Comment:
These two lines are auto-migrations by `flutter` that have just never been checked in.